docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-defining-a-stage-proc.adoc

```diff
@@ -1,2 +1,28 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-defining-a-stage-proc']
+= Defining a stage
+
+A stage can be modeled in BPMN2 using the case designer. Stages are a way of grouping related tasks in a way that clearly define activities that must complete before the next stage of the case commences. For example, the IT Orders case definition described in <<case-management-designing-a-case-proc>> can also be defined using stages in the following way:
```

Comment:
> clearly define activities that must complete before

they don't have to complete. since it's an ad hoc activities defined there might be but does not have to be executed

Reply:
added caveat: "if the stage is activated".